src/sqlodbc/opensearch_communication.h

```diff
@@ -81,6 +81,7 @@ class OpenSearchCommunication {
     void SetErrorDetails(std::string reason, std::string message,
                          ConnErrorType error_type);
     void SetErrorDetails(ErrorDetails details);
+    bool isServerless();
```

Comment:
Probably should be marked const since it shouldn't mutate state. Capital 'i' as well.

Reply:
Overtaken by subsequent changes. Has been replaced with non-`const` `SetIsServerless` method.